lib/internal/util.js

```diff
@@ -431,12 +432,34 @@ function createDeferredPromise() {
   return { promise, resolve, reject };
 }
 
+let inspect = null;
+function lazyLoadInspect() {
+  if (inspect !== null) {
+    return inspect;
+  }
+  return inspect = require('util').inspect;
+}
+
+function customInspect(self, obj, depth, options) {
+  const name = self.constructor.name;
+  if (depth < 0)
+    return name;
```

Comment:
To avoid regressing `test/parallel/test-eventtarget-memoryleakwarning.js`:
```suggestion
  if (depth < 0)
    return self;

  const name = self.constructor.name;
```

This ensures that when `depth < 0`, the formatting is handled by `inspect`, which only returns the object’s tag name.